<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDifficultGame</class>
 <widget class="QWidget" name="SelectDifficultGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1086</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="__wrapper" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>90</number>
    </property>
    <property name="leftMargin">
     <number>60</number>
    </property>
    <property name="topMargin">
     <number>65</number>
    </property>
    <property name="rightMargin">
     <number>60</number>
    </property>
    <property name="bottomMargin">
     <number>130</number>
    </property>
    <item>
     <widget class="QLabel" name="__title_text">
      <property name="font">
       <font>
        <family>Marmelad</family>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберите режим игры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="__content_box" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>61</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>62</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>31</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="__l_cards_box">
         <property name="spacing">
          <number>33</number>
         </property>
         <property name="leftMargin">
          <number>80</number>
         </property>
         <property name="rightMargin">
          <number>80</number>
         </property>
         <item>
          <widget class="QWidget" name="__card_1" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>374</width>
             <height>456</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <widget class="QWidget" name="__w_text_box_1" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>349</y>
              <width>374</width>
              <height>90</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>374</width>
              <height>90</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_text_1">
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>35</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Простой</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="__card_content_1" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>334</width>
              <height>456</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>334</width>
              <height>456</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>88</number>
             </property>
             <property name="topMargin">
              <number>145</number>
             </property>
             <property name="rightMargin">
              <number>88</number>
             </property>
             <property name="bottomMargin">
              <number>228</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_content_text_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>158</width>
                 <height>83</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>50</pointsize>
                </font>
               </property>
               <property name="text">
                <string>4 x 4</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <zorder>__card_content_1</zorder>
           <zorder>__w_text_box_1</zorder>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="__card_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>374</width>
             <height>456</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <widget class="QWidget" name="__w_text_box_2" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>349</y>
              <width>374</width>
              <height>90</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>374</width>
              <height>90</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_text_2">
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>35</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Средний</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="__card_content_2" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>334</width>
              <height>456</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>334</width>
              <height>456</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>51</number>
             </property>
             <property name="topMargin">
              <number>145</number>
             </property>
             <property name="rightMargin">
              <number>51</number>
             </property>
             <property name="bottomMargin">
              <number>228</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_content_text_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>232</width>
                 <height>83</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>50</pointsize>
                </font>
               </property>
               <property name="text">
                <string>8 x 8</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <zorder>__card_content_2</zorder>
           <zorder>__w_text_box_2</zorder>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="__card_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>374</width>
             <height>456</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <widget class="QWidget" name="__w_text_box_3" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>349</y>
              <width>374</width>
              <height>90</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>374</width>
              <height>90</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_text_3">
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>35</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Сложный</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="__card_content_3" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>334</width>
              <height>456</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>334</width>
              <height>456</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>51</number>
             </property>
             <property name="topMargin">
              <number>145</number>
             </property>
             <property name="rightMargin">
              <number>51</number>
             </property>
             <property name="bottomMargin">
              <number>228</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_content_text_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>232</width>
                 <height>83</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>50</pointsize>
                </font>
               </property>
               <property name="text">
                <string>16 x 16</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <zorder>__card_content_3</zorder>
           <zorder>__w_text_box_3</zorder>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="__card_4" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>374</width>
             <height>456</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <widget class="QWidget" name="__w_text_box_4" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>349</y>
              <width>374</width>
              <height>90</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>374</width>
              <height>90</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_text_4">
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>35</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Свой</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="__card_content_4" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>334</width>
              <height>456</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>334</width>
              <height>456</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>92</number>
             </property>
             <property name="topMargin">
              <number>145</number>
             </property>
             <property name="rightMargin">
              <number>93</number>
             </property>
             <property name="bottomMargin">
              <number>228</number>
             </property>
             <item>
              <widget class="QLabel" name="__card_content_text_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>158</width>
                 <height>83</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Marmelad</family>
                 <pointsize>50</pointsize>
                </font>
               </property>
               <property name="text">
                <string>? x ?</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <zorder>__card_content_4</zorder>
           <zorder>__w_text_box_4</zorder>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>28</number>
         </property>
         <item>
          <spacer name="__h_spacer_button">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="__button_back">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Marmelad</family>
             <pointsize>40</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Вернуться</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
